hawk/app/assets/stylesheets/shared/_misc.scss

```diff
@@ -32,3 +32,30 @@
     border-radius: 20px;
   }
 }
+
+.diff_sub {
+  background: #fdd;
+}
+
+.diff_add {
+  background: #dfd;
```

Comment:
Color literals like `#dfd` should only be used in variable declarations; they should be referred to via variable everywhere else.